File: Backend/proyecto/src/main/java/com/admindb/proyecto/modelo/permisos/PermisoColumna.java
package com.admindb.proyecto.modelo.permisos;


import javax.persistence.Column;
import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;

import lombok.Getter;
import lombok.Setter;



@Getter
@Setter
@Entity
@Table(name = "dba_col_privs")
public class PermisoColumna {
    
    @EmbeddedId
    private PermisoColumnaId id;

    @Column(name = "owner")
    private String propietario;

    @Column(name = "grantee")
    private String otorgado;
    
}

File: Frontend/frontend-adminbd/src/app/common/permiso-columna.ts
import { PermisoColumnaId } from "./permiso-columna-id";

export class PermisoColumna {
    id: PermisoColumnaId;
    propietario: string;

    constructor(id: PermisoColumnaId, propietario: string){
        this.id=id ;
        this.propietario = propietario;
    }
}
